idea/testData/inspectionsLocal/suspiciousCollectionReassignment/mutableList.kt

```diff
@@ -0,0 +1,7 @@
+// PROBLEM: none
+// ERROR: Assignment operators ambiguity: <br>public operator fun <T> Collection<Int>.plus(element: Int): List<Int> defined in kotlin.collections<br>@InlineOnly public inline operator fun <T> MutableCollection<in Int>.plusAssign(element: Int): Unit defined in kotlin.collections
+// WITH_RUNTIME
+fun test() {
+    var list = mutableListOf(1)
+    <caret>list += 2
```

Comment:
Just as an idea: why don't fix it by changing `var` to `val`?

Reply:
Created: https://youtrack.jetbrains.com/issue/KT-26236